lib/auth/webauthncli/fido2.go

```diff
@@ -54,8 +54,12 @@ const (
 	fido2DeviceTimeout = 30 * time.Second
 
 	// Operation retry interval.
-	// Keep it less frequent than 2Hz / 0.5s.
+	// Keep it less frequent than 5Hz / 0.2s.
```

Comment:
Is this 5Hz frequency coming from some source or was it tested?

Reply:
It's based on https://developers.yubico.com/libfido2/Manuals/fido_dev_get_touch_status.html#CAVEATS. That doesn't specifically mentions the actual retries we do here, but it seemed reasonable to me to avoid going faster than that.